### PR TITLE
Switched to fetching images from macaulay library and added image precaching

### PR DESCRIPTION
**Pull request by SuperStormer**

### CODE REVIEW

File: cogs/check.py [hunk 1]
```diff
@@ -33,40 +37,44 @@ def __init__(self, bot):
 
         await channel_setup(ctx)
         await user_setup(ctx)
-
-        global achievement
```
Comment (tctree333) [inline cogs/check.py]: Will achievement still be accessible locally if this line is removed?

File: cogs/score.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # score.py | commands to show score related things
-# Copyright (C) 2019  EraserBird, person_v1.32
+# Copyright (C) 2019  EraserBird, stats_v1.32, hmmm
```
Comment (tctree333) [inline cogs/score.py]: May want to double check this line

File: cogs/score.py [hunk 3]
```diff
@@ -93,39 +110,44 @@ def __init__(self, bot):
...
-                user = self.bot.get_user(int(leaderboard_list[x][0]))
+                user = self.bot.get_user(int(stats[0]))
                 if user is None:
                     user = "**Deleted**"
                 else:
                     user = f"**{user.name}#{user.discriminator}**"
             else:
-                user = f"<@{str(leaderboard_list[x][0])[2:-1]}>"
-            leaderboard += f"{str(x+1)}. {user} - {str(int(leaderboard_list[x][1]))}\n"
+                user = f"<@{str(leaderboard_list[i][0])[2:-1]}>"
```
Comment (tctree333) [inline cogs/score.py]: Not entirely sure what’s happing here

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,5 @@
 discord.py==1.2.3
 wikipedia==1.4.0
-google_images_download==2.8.0
 aiohttp==3.5.4
 eyeD3==0.8.10
 redis==3.3.5
```
Comment (tctree333) [inline requirements.txt]: Should probably add all the other modules you’re using unless they’re included in the ones here or built in

**Comment by tctree333** [comment]
Should probably have left these comments on the review but eh ¯\_(ツ)_/¯

File: requirements.txt (hunk 4 above)
Comment (SuperStormer) [reply requirements.txt]: the only library i used in my code was aiohttp

File: cogs/check.py (hunk 1 above)
Comment (SuperStormer) [reply cogs/check.py]: global statements are only needed if you're assigning to a global value(which you're not)

File: cogs/score.py (hunk 3 above)
Comment (SuperStormer) [reply cogs/score.py]: that was rewriting your range-len loop into an enumerate loop

**Comment by SuperStormer** [comment]
latest commit _shouldn't_ break anything and also fixes the copyright issue

File: cogs/check.py (hunk 1 above)
Comment (tctree333) [reply cogs/check.py]: Ah, ok!

File: cogs/score.py (hunk 3 above)
Comment (tctree333) [reply cogs/score.py]: Just wanted to check if the line was supposed to be changed

File: requirements.txt (hunk 4 above)
Comment (tctree333) [reply requirements.txt]: Ah ok

**Pull request merged**